### PR TITLE
refactor: import Quandl and Coinbase feeds from d3fc-financial-feed

### PR DESCRIPTION
**Pull request by tlsim**

### CODE REVIEW

File: src/data/data.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import feed from './feed/feed';
+import { coinbase, quandl } from 'd3fc-financial-feed';
```
Comment (chrisprice) [inline src/data/data.js]: Not a big deal but I think you can `import * as feed from 'd3fc-financial-feed'`

**Comment by ColinEberhardt** [comment]
This looks good to me :-)

Although I think the website should be updated as part of this change, removing the information which is how duplicated.

**Comment by tlsim** [comment]
@ColinEberhardt site now updated.

**Comment by ColinEberhardt** [comment]
Nice job ... +10 / -325 good diff statistics :-)

**Pull request merged**